src/include/sof/audio/component.h

```diff
@@ -165,13 +165,20 @@ enum {
 #define comp_cl_dbg(drv_p, __e, ...) LOG_DBG(__e, ##__VA_ARGS__)
 
 /* device level tracing */
-#define comp_err(comp_p, __e, ...) LOG_ERR(__e, ##__VA_ARGS__)
 
-#define comp_warn(comp_p, __e, ...) LOG_WRN(__e, ##__VA_ARGS__)
+#define __COMP_FMT "comp:%u.%u "
```

Comment:
Hmm, @abonislawski  a slight beauty issue is that this is common format for IPC3 and IPC4. Maybe I'll actually ifdef the format part. V2 coming...

Reply:
@abonislawski Hmm, it seems pretty much all kernel logs (FYI @ujfalusi @RanderWang @ranj063 ) use a "%u" to print pipeline-ids also for IPC4. 

I now made a compromise that I print the pipeline-id as "%u" and use hex-output for the component id (which is a bitmask in IPC4).

Logs now look:
```
[   30.376000] <inf> pipe: pipe:2.0 pipe reset
[   30.376000] <inf> dai_comp: comp:2.40001 dai_reset()
[   30.376000] <inf> dw_dma: dw_dma_stop(): dma 0 channel 0 stop
[   30.376000] <inf> dw_dma: dw_dma_channel_put(): dma 0 channel 0 put
[   30.376000] <inf> ipc: ipc: new cmd 0x93010003|0x0
[   30.376000] <inf> pipe: pipe:1.0 pipe trigger cmd 2
[   30.376000] <err> copier: comp:1.40000 failed to find dai comp or sink pipeline not running.
```

Reply:
Yes, it is a bitmask, I dont remember which format was used in kernel but if there are IDs like 393217 then please update kernel too, at least for IPC4 if not everywhere.

btw. you can go with %#x so it will be obvious what we are looking at